net/znc/files/znc.init

```diff
@@ -12,7 +15,6 @@ DISABLED=
 
 RUNAS_USER=
 RUNAS_GROUP=
-RUNAS_SHELL=
```

Comment:
Please don't drop features silently. This was the place where I stopped my try at the conversion, because I couldn't at a first glance how to convert this to procd (or how it is used) so I chased other targets.

@Xotic750 can you maybe describe where this feature is needed, and how one can test whether it works?